docs/fundamentals/code-analysis/code-style-rule-options.md

```diff
@@ -30,151 +27,246 @@ Each of these subcategories defines its own syntax for specifying options. For m
 To help you get started, here's an example *.editorconfig* file with the default options.
 
 > [!NOTE]
-> The concise syntax seen in this file, where the severity is specified after the option value for language rules, for example, `dotnet_style_readonly_field = true:suggestion`, is only fully understood by Visual Studio. Compilers don't understand the `:<severity-level>` suffix and it is ignored if you compile your code from the command line. To set severity in a way that's understood by both Visual Studio and the C# and Visual Basic compilers, set the severity on the rule that contains the option by using the following syntax:
->
-> `dotnet_diagnostic.<rule-ID>.severity = <severity-level>`
->
-> For more information, see [Option format](style-rules/language-rules.md#option-format).
+> If you're using .NET 8 or an earlier version, the syntax that's used to specify severity in this example file might not be respected at build time. In that case, you can set the severity by using the rule ID-based severity configuration syntax for analyzers instead. For more information, see [Option format](style-rules/language-rules.md#option-format).
```

Comment:
What about when there's no option for a rule? I've spent quite some time to get our build/format to fail for an `IDE0052` which is showing as a problem in VS Code with the C# extensions, but refuses to surface via CLI/CI builds, .NET 8 (LTS). I found this note and wondered if it might unlock it for us, but from [the docs](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0052) "This rule has no associated code-style options".